packages/docs/components/docs-content/api/va-modal/VaModalApi.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
We probably want some padding here:
![image](https://user-images.githubusercontent.com/5394573/81243626-0a52c480-9019-11ea-9d28-7eb0f890b2f2.png)